fud/fud/utils.py

```diff
@@ -192,3 +192,23 @@ def shell(cmd, stdin=None, stdout_as_debug=False):
         else:
             raise errors.StepFailure(cmd, "No stdout captured.", "No stderr captured.")
     return stdout
+
+
+def transparent_shell(cmd, stdout_as_debug=False):
+    """
+    Runs `cmd` in the shell. Does not capture output or input. Does nothing
+    fancy and returns nothing
+    """
+    if isinstance(cmd, list):
+        cmd = " ".join(cmd)
+
+    if stdout_as_debug:
+        cmd += ">&2"
```

Comment:
Instead of using `shell=True` and concatenating the command to a string so you can do shell syntax for redirection, you could ask `Popen` to do the stream redirection explicitly. In particular, not 100% sure about this, but I think it should work to do `Popen(..., stdout=sys.stderr)`.

Reply:
Good point, I just copied from the existing shell def. Upon further consideration I don't think it makes sense for the transparent shell to support this since by design it isn't capturing anything

Reply:
Got it! Oops; didn't realize this existed in the old shell utility too. Anyway, yeah, all the more reason to drop it and `shell=True` altogether!